### PR TITLE
Issue-613/Fix missing b2xIdentityUserFlow-id and related entities

### LINKED ISSUE #613: Missing Data: {b2xIdentityUserFlow-id}
Test Cases:
- [ ] get-b2xuserflow-list-identityproviders-csharp-V1-executes
- [ ] get-b2xuserflows-apiconnectorconfiguration-csharp-V1-executes
- [x] get-b2xuserflows-csharp-V1-executes
- [ ] get-identityuserflowattributeassignment-2-csharp-V1-executes
- [ ] get-userflowlanguageconfiguration-2-csharp-V1-executes
- [ ] get-identityuserflowattributeassignment-expand-2-csharp-V1-executes
- [ ] identityuserflowattributeassignment-getorder-csharp-V1-executes

### PR DESCRIPTION
**Pull request by fey101**
- Fixes #613 
- Excludes 2 PSScriptAnalyzer rules project wide as they do not add much value as discussed in this 2 threads:https://github.com/PowerShell/PSScriptAnalyzer/issues/293 and https://github.com/PowerShell/PSScriptAnalyzer/issues/1118
- Removes fixed tests from CsharpKnownIssuesList

### CODE REVIEW

File: PSScriptAnalyzerSettings.psd1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This is the settings file that configures rules for linting of powershell scripts
+# across the entire raptor project
+@{
+    # Do not analyze the following rules.
+    # Note: if a rule is in both IncludeRules and ExcludeRules, the rule
+    # will be excluded.
+    ExcludeRules = @(
+        'PSAvoidUsingWriteHost',
+        'PSUseBOMForUnicodeEncodedFile'
```
[the remark this answers is not in the document]
Comment (fey101) [reply PSScriptAnalyzerSettings.psd1:9]: VSCode does not save files with the BOM character by default.

File: PSScriptAnalyzerSettings.psd1 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# This is the settings file that configures rules for linting of powershell scripts
+# across the entire raptor project
+@{
+    # Do not analyze the following rules.
+    # Note: if a rule is in both IncludeRules and ExcludeRules, the rule
+    # will be excluded.
+    ExcludeRules = @(
+        'PSAvoidUsingWriteHost',
```
[the remark this answers is not in the document]
Comment (fey101) [reply PSScriptAnalyzerSettings.psd1:8]: I think writing a wrapper is a hack that may cause issues like https://www.experts-exchange.com/questions/28958490/PowerShell-format-table-with-Host-UI-RawUI-ForegroundColor-issues.html yet write-host works just fine for our scenarios. But if you feel convinced about using the wrapper, I can remove the exclusion for write-host.

**Review by georgend — APPROVED** [review]

**Review by georgend — APPROVED** [review]

File: report/V1-csharp-execution-known-issues-report.html [hunk 3]
```diff
@@ -18,7 +18,7 @@
                 labels: ['MissingData','MissingPermissionScope','Documentation','SDKkiotaTriage','NeedsAnalysis','SDK','Service','HTTP','ProtectedAPI','Permissions'],
                 datasets: [{
                     label: '# of issues',
-                    data: [78,59,6,4,3,3,3,2,2,1],
+                    data: [66,59,6,4,3,3,3,2,2,1],
```
Comment (zengin) [inline report/V1-csharp-execution-known-issues-report.html:21]: 🎉

File: scripts/TenantSetup/TenantHttpData/userFlows/apiConnector.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "displayName":"raptorcallbacks_test_api",
+    "targetUrl":"https://raptorcallbacks.azurewebsites.net/api",
```
Comment (zengin) [inline scripts/TenantSetup/TenantHttpData/userFlows/apiConnector.json:3]: is this just a placeholder URL or did we spin up something?

File: scripts/TenantSetup/TenantHttpData/userFlows/b2xIdentityUserFlow.ps1 [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    $b2xIdentityUserFlow = Request-DelegatedResource -Uri "identity/b2xUserFlows" -Method "POST" -Body $b2xuserFlowData -ScopeOverride "IdentityUserFlow.ReadWrite.All"
+}
+$b2xIdentityUserFlow.id
+$identifiers.b2xIdentityUserFlow._value = $b2xIdentityUserFlow.id
+
+$identityUserFlowAttributeAssignment = Request-DelegatedResource -Uri "identity/b2xUserFlows/$($b2xIdentityUserFlow.id)/userAttributeAssignments?`$top=1" -ScopeOverride "IdentityUserFlow.Read.All"
+$identityUserFlowAttributeAssignment.id
+$identifiers.b2xIdentityUserFlow.identityUserFlowAttributeAssignment._value = $identityUserFlowAttributeAssignment.id
+
+$userLanguage = Request-DelegatedResource -Uri "identity/b2xUserFlows/$($b2xIdentityUserFlow.id)/languages?`$filter=displayName eq 'français'" -ScopeOverride "IdentityUserFlow.Read.All"
```
Comment (zengin) [inline scripts/TenantSetup/TenantHttpData/userFlows/b2xIdentityUserFlow.ps1:33]: userLanguage: 🇫🇷
cc @baywet :)

**Review by zengin — APPROVED** [review]

File: scripts/TenantSetup/TenantHttpData/userFlows/apiConnector.json (hunk 4 above)
Comment (fey101) [reply scripts/TenantSetup/TenantHttpData/userFlows/apiConnector.json:3]: It's been there all along. It's the same one we use for our notifications callbackurl in subscriptions

**Pull request merged**